plugins/outputs/mongodb/mongodb.go

```diff
@@ -145,6 +177,8 @@ func (s *MongoDB) Init() error {
 			AuthMechanism: "MONGODB-X509",
 		}
 		s.clientOptions.SetAuth(credential)
+	default:
+		return fmt.Errorf("unsupported authentication type %q; supported types are: SCRAM, PLAIN, X509", s.AuthenticationType)
```

Comment:
Please keep those errors brief. The config should mention available options! This way we reduce redundancy and avoid trouble when adding more types (or removing some) in the future.

```suggestion
		return fmt.Errorf("unsupported authentication type %q", s.AuthenticationType)
```